ldes-fragmentisers/ldes-fragmentisers-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentationService.java

```diff
@@ -65,7 +65,7 @@ public void handleViewInitializationEvent(ViewNeedsRebucketisationEvent event) t
 
 	@Scheduled(cron = FRAGMENTATION_CRON)
 	public void scheduledJobLauncher() throws JobInstanceAlreadyCompleteException, JobExecutionAlreadyRunningException, JobParametersInvalidException, JobRestartException {
-		if (shouldTriggerBucketisation.get() && !isJobRunning(BUCKETISATION_JOB) && !isJobRunning(REBUCKETISATION_JOB)) {
+		while (shouldTriggerBucketisation.get() && !isJobRunning(BUCKETISATION_JOB) && !isJobRunning(REBUCKETISATION_JOB)) {
 			launchJob(bucketiseJob, new JobParameters());
```

Comment:
Should most definitely not be a while but an if statement. The repetitiveness is caused by the CRON already